src/main/kotlin/lotto/exceptions/LottoException.kt

```diff
@@ -37,4 +37,19 @@ sealed class LottoException(msg: String) : IllegalArgumentException("[ERROR]: $m
         "Invalid lotto number: '$value'. Expected a number in the range of ${LottoNumber.MINIMUM_NUMBER} " +
             "to ${LottoNumber.MAXIMUM_NUMBER}.",
     )
+
+    class InvalidManualTicketsSizeFormatException(input: String) : LottoException(
+        "Invalid manual tickets number format: $input",
+    )
+
+    class InvalidManualTicketsNumberException(amount: Int) : LottoException(
+        "Invalid amount of manual tickets: $amount: should be less or equal to the amount of purchased tickets.",
+    )
+
+    class InvalidManualTicketsException() : LottoException("Invalid manual tickets generations.")
```

Comment:
```suggestion
    class InvalidManualTicketsException : LottoException("Invalid manual tickets generations.")
```

Kotlin allows us to keep things simple 😋